### ISSUE TITLE
Refactor OptionBox/CheckBox

### ISSUE THREAD

**Issue by ianharrigan** [ianharrigan/haxeui#250]
These are overly complicated compound controls and as such are hard to style and understand

**Comment by roughee** [comment]
Maybe you could give an example how to style Text of OptionBox?
Also, I tried to add mouseOver/mouseOut event listeners on text, but they don't seem to work.

**Comment by ianharrigan** [comment]
Agreed, version 2 will have a new optionbox/checkbox which should be simpler
